### PR TITLE
Pre-release updates

### PR DESCRIPTION
**Pull request by fortuna**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,9 +33,13 @@ The system comprises the following components:
 ## Code Prerequisites
 
 In order to build and run the code, you need the following installed:
-  - [Node](https://nodejs.org/) (version `16.13.0`)
-  - [NPM](http://npmjs.org/) (version `8.1.0`)
-  - [Wine](https://www.winehq.org/download), if you would like to generate binaries for Windows.
+  - [Node](https://nodejs.org/en/download/) LTS (version `16.13.0`)
+  - [NPM](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) (version `8.1.0`)
+  - Manager-specic
+    - [Wine](https://www.winehq.org/download), if you would like to generate binaries for Windows.
```
Comment (jyyi1) [inline README.md:39]: Do we really need Wine here?

Comment (fortuna) [reply README.md:39]: I don't know. Maybe not? But I'd address that in a different PR.

Comment (jyyi1) [reply README.md:39]: 👍

**Review by jyyi1 — APPROVED** [review]

**Review by daniellacosse — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -33,9 +33,13 @@ The system comprises the following components:
 ## Code Prerequisites
 
 In order to build and run the code, you need the following installed:
-  - [Node](https://nodejs.org/) (version `16.13.0`)
-  - [NPM](http://npmjs.org/) (version `8.1.0`)
-  - [Wine](https://www.winehq.org/download), if you would like to generate binaries for Windows.
+  - [Node](https://nodejs.org/en/download/) LTS (version `16.13.0`)
```
Comment (daniellacosse) [inline README.md:36]: might be worth calling out the code name `lts/gallium`

Comment (fortuna) [reply README.md]: Done

**Pull request merged**